packages/checkout/registry/index.js

```diff
@@ -53,45 +52,13 @@ const getCheckoutFilters = ( filterName ) => {
 export const __experimentalApplyCheckoutFilter = ( {
 	filterName,
 	defaultValue,
-	extensions = {},
-	arg: _arg = null,
-	validation: _validation = () => true,
+	extensions,
+	arg = null,
+	validation = () => true,
```

Comment:
`() => true` will be considered a different function on each render and that will cause `useMemo()` to always rerun. That's not an issue now because all ours filters specify a `validation` argument, but I think it's still worth fixing it.